playbooks/roles/xqueue/defaults/main.yml

```diff
@@ -114,3 +114,5 @@ xqueue_debian_pkgs:
   - libgeos-dev
   # Needed to be able to create the xqueue mysqldb.
   - python-mysqldb
+
+nginx_template_directory: "edx/app/nginx/sites-available"
```

Comment:
Actually, don't do that yet. I think we need that because the XQueue play still pulls in values from edxapp, so this may be needed to override that..